nixos/modules/module-list.nix

```diff
@@ -72,6 +72,8 @@
   ./hardware/hackrf.nix
   ./hardware/i2c.nix
   ./hardware/infiniband.nix
+  ./hardware/inputmodule.nix
+  ./hardware/iosched.nix
```

Comment:
Huh, why did the backport action add inputmodule?

cc CI maintainers @Mic92 @zowoq @infinisil

Reply:
Okay wild I actually reproduced this bug in plain git; this is git's doing o.O